pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -246,6 +246,10 @@ func (b *awsElasticBlockStoreBuilder) IsReadOnly() bool {
 	return b.readOnly
 }
 
+func (b *awsElasticBlockStoreBuilder) SupportsSELinux() bool {
+	return !b.IsReadOnly()
```

Comment:
This needs to sync with https://github.com/kubernetes/kubernetes/pull/15352/files#r42777530 wherein I propose (reversing myself from previous, sorry) that volumes should not consider read-only-ness here, but the caller should.